includes/wcs-cart-functions.php

```diff
@@ -108,7 +108,7 @@ function wcs_cart_totals_shipping_html() {
 							<?php wcs_cart_print_shipping_input( $recurring_cart_package_key, $shipping_method ); ?>
 							<?php do_action( 'woocommerce_after_shipping_rate', $shipping_method, $recurring_cart_package_key ); ?>
 							<?php if ( ! empty( $show_package_details ) ) : ?>
-								<?php echo '<p class="woocommerce-shipping-contents"><small>' . esc_html( $package_details ) . '</small></p>'; ?>
+								<?php echo '<p class="woocommerce-shipping-contents"><small>' . wp_kses_post( apply_filters( 'woocommerce_cart_item_name', $package_details ) ) . '</small></p>'; ?>
```

Comment:
I used the same approach used by the product name on the cart page [here](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/cart/cart.php#L97).